BLSQ/iaso
# POLIO-903: budget tab modality of payment dropdown

Budget tab has only one select field for payment mode but the updated version includes a payment mode for UNICEF and an other one for WHO

<img width="1300" alt="Capture d’écran 2023-04-13 à 14 32 03" src="https://user-images.githubusercontent.com/25134301/231759224-6834bb0d-0c6e-4476-b16f-b9dcdad9a536.png">

![mockup](https://user-images.githubusercontent.com/25134301/231758977-bbb72c54-ef8a-4e87-8949-c33cdda28a3e.png)

Related JIRA tickets : [POLIO-903](https://bluesquare.atlassian.net/browse/POLIO-903?atlOrigin=eyJpIjoiYWY0MzY4ZjQ4MjlkNGE1ZWE2ZTZhNTkxNDMxMTJkNDQiLCJwIjoiaiJ9)

## Self proofreading checklist


## Changes

- refactor `PaymentField` component with typescript
- new translations
- update constant types to add new payment mode "Mobile Money"
- "Mobile Money" as payment choice 

## How to test

- Go to Campaigns
- Create or edit a campaign
- Go to Budget Tab :  Under the Funds Release column you should see two payment mode select fields, one for UNICEF and one for WHO

## Print screen / video
<img width="1387" alt="Capture d’écran 2023-04-13 à 14 38 28" src="https://user-images.githubusercontent.com/25134301/231760808-a7481d83-c797-4d07-9476-80e0ccadb872.png">

[POLIO-903]: https://bluesquare.atlassian.net/browse/POLIO-903?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
